src/main/java/com/conveyal/analysis/persistence/Persistence.java

```diff
@@ -36,7 +35,6 @@
     private static DB db;
 
     public static MongoMap<Modification> modifications;
-    public static MongoMap<Project> projects;
```

Comment:
I discovered this is still used by the CacheWarmer. Might be better to keep the Project class and this `MongoMap<Projects>` - removing them will break the cluster build and require refactors there.

Reply:
Added it back 👍